src/plugins/walletConnectToDapps/v2/typeGuards.ts

```diff
@@ -44,10 +44,7 @@ export const isTransactionParams = (
   !!transaction?.from &&
   !!transaction?.to &&
   !!transaction?.data &&
-  !!transaction?.value &&
-  !!transaction?.gasLimit &&
-  !!transaction?.gasPrice &&
-  !!transaction?.nonce
+  ((!!transaction?.gasLimit && !!transaction?.gasPrice) || !!transaction?.gas)
```

Comment:
The guts of this:
- removes `nonce` which isn't actually a requirement
- expects either `gasLimit` and `gasPrice`, or only `gas`, so we stop throwing

Reply:
I'm happy we are now using `!!` over `Boolean` 😁